src/test/scala/com/advancedtelematic/director/http/AdminResourceSpec.scala

```diff
@@ -196,6 +197,14 @@ trait AdminResourceSpec extends DirectorSpec with KeyGenerators with DeviceRegis
 
     fetchRootOk(1).signed shouldBe fetchRootOk.signed
   }
+
+  testWithNamespace("multi_target_update/updateId gives all the MTUs in the namespace") { implicit ns =>
+    val newMTURequest = GenMultiTargetUpdateRequest.generate
+    val updateId = createMultiTargetUpdateOK(newMTURequest)
+
+    val foundUpdates = findByUpdate(updateId)
+    foundUpdates.map(_.id).distinct.loneElement should be(updateId)
```

Comment:
Didn't know about `loneElement`, nice!

Reply:
All that time I spent reading docs finally payed off! haha